docs/sources/mimir/operators-guide/tools/mimirtool.md

```diff
@@ -336,6 +336,8 @@ mimirtool rules diff <file_path>...
 
 The format of the file is the same format as shown in [rules load](#load-rule-group).
 
+To restrict the affected namespaces, use the `--namespaces` and `--ignore-namespaces` parameters as well as their RegEx variants `--namespaces-regex` and `--ignore-namespaces-regex`.
```

Comment:
Other flags in this file are listed in a table. See the "Configuration" section of the "prepare" command. Can you transform these two in this way? Same comment applies to the docs for the `sync` command.